doc/operations/recovery.rst

```diff
@@ -145,6 +145,94 @@ Which of these two paths is taken is noted in the `public:ccf.internal.last_reco
       ...
     $ /opt/ccf/bin/js_generic --config /path/to/config/file
 
+Self-Healing-Open recovery
+--------------------------
+
```

Comment:
Note a note or warning to say this is experimental still. Also a changelog entry definitely.